src/xcode/ENA/ENA/Source/Models/Exposure/ExposureManager.swift

```diff
@@ -192,7 +195,8 @@ final class ENAExposureManager: NSObject, ExposureManager {
 		.init(
 			authorized: type(of: manager).authorizationStatus == .authorized,
 			enabled: manager.exposureNotificationEnabled,
-			status: manager.exposureNotificationStatus
+			status: manager.exposureNotificationStatus,
+			authorizationStatus: type(of: manager).authorizationStatus
```

Comment:
no need to inject it:  just use ENManager.authorizationStatus where you need it.. its a static class variable

Reply:
Are we sure we want to access ```ENManager``` directly here and skip ```ENAExposureManager```?

Reply:
Yes

Reply:
I mean, at this point I'm totally fine with both options, it's only for the sake of someone else doing the review and asking why I'm skipping ```ENAExposureManager``` and call ```ENManager``` (the Apple framework) directly.

Reply:
We should do it everywhere in the same way and not introduce new complexity ... In statehandler we also call ENManager

Reply:
Ok, will change it then. I think we introduced complexity at the point we started calling ```ENManager``` directly, instead of channelling the call through ```ENAExposureManager```. But nevermind, I agree that my PR shouldn't introduced further complexity.

Reply:
I'd changed this, but after that I've simplified the logic to adjust it to the current state of development, so this is not needed anymore. 

Please check [here](https://github.com/corona-warn-app/cwa-app-ios/pull/517/commits/e7ed188224d3aff072ed466b6c0b274e8290ee9c).